prow_config.yaml

```diff
@@ -4,3 +4,20 @@ workflows:
   - app_dir: kubeflow/testing/workflows
     component: workflows
     name: unittests
+
+  - app_dir: kubeflow/kubeflow/testing/workflows
```

Comment:
I assume this is what you want to delete?

Reply:
I think if you wanted to keep this you could. Its not a bad idea to leave this here to verify that presubmits skip files if they aren't changed.

Just add a comment explaining what this is for.  And change app_dir to   some/nonexistent/appdir

In the comment explain that this verifies that workflow is skipped because the test would fail since it doesn't exist.

Reply:
Done.